### PR TITLE
ASoC: SOF: Fix uninitialized usage

### PR DESCRIPTION
**Pull request by cujomalainey**
Courtesy of clang checks

### CODE REVIEW

File: sound/soc/sof/ipc4-topology.c [hunk 1]
```diff
@@ -882,10 +882,8 @@ static int sof_ipc4_widget_setup_comp_process(struct snd_sof_widget *swidget)
 	/* allocate memory for base config extension if needed */
 	if (process->init_config == SOF_IPC4_MODULE_INIT_CONFIG_TYPE_BASE_CFG_WITH_EXT) {
 		struct sof_ipc4_base_module_cfg_ext *base_cfg_ext;
-		u32 ext_size;
-
-		ext_size += struct_size(base_cfg_ext, pin_formats,
-					swidget->num_input_pins + swidget->num_output_pins);
+		u32 ext_size = struct_size(base_cfg_ext, pin_formats,
+						swidget->num_input_pins + swidget->num_output_pins);
```
Comment (plbossart) [inline sound/soc/sof/ipc4-topology.c:886]: I think the commit title needs to be 

fixup! ASoC: SOF: ipc4-topology: Add support for base config extension

and the Fixes tag will be dropped as well, we only use it for patches that have landed in Mark Brown's tree.

**Review by plbossart — DISMISSED** [review]

File: sound/soc/sof/ipc4-topology.c (hunk 1 above)
Comment (cujomalainey) [reply sound/soc/sof/ipc4-topology.c:886]: Ah I didnt check marks, will update title

Comment (ranj063) [reply sound/soc/sof/ipc4-topology.c:886]: @cujomalainey can you please try ```git commit --fixup``` with the hash for the commit in Pierre's comment.

Comment (cujomalainey) [reply sound/soc/sof/ipc4-topology.c:886]: Did not know that was a command, thanks for the pointer

Comment (ujfalusi) [reply sound/soc/sof/ipc4-topology.c:886]: @cujomalainey, can you update the commit title to:
`fixup! ASoC: SOF: ipc4-topology: Add support for base config extension`
so we will know which patch this should be applied to?

I wonder how things worked with this bug... Thanks for finding it!!!

**Review by plbossart — APPROVED** [review]

**Comment by plbossart** [comment]
@ranj063 I'll let you merge if you're ok with the fix

**Review by ranj063 — APPROVED** [review]

**Comment by ranj063** [comment]
> @ranj063 I'll let you merge if you're ok with the fix

sign-off is missing but its a fixup so we can let it go

**Comment by cujomalainey** [comment]
> > @ranj063 I'll let you merge if you're ok with the fix
> 
> sign-off is missing but its a fixup so we can let it go

Just tried signing off on another fixup, doesn't seem to be allowed

**Pull request merged**